albums/complete_theme_album.md

```diff
@@ -1,408 +1,380 @@
-Darling in the Franxx
+AzurLane (Azur Lane)
 ---
 
-### Zero Two
+### Essex
+![Essex (Dark)](../screenshots/azurLane/essex_dark_code.png)
+![Essex (Dark, Plain)](../screenshots/azurLane/essex_dark_plain_code.png)
 
-![zero two code](../screenshots/franxx/zero_two_dark_code.png)
-![zero two code](../screenshots/franxx/zero_two_light_code.png)
 
-Chuunibyou Love and Other Delusions
+Blend S (BLEND-S)
 ---
 
-### Takanashi Rikka
-
-![rikka_dark code](../screenshots/chuunibyou/rikka_dark_code.png)
+### Maika Sakuranomiya
+![Maika Sakuranomiya (Dark)](../screenshots/blendS/maika_dark_code.png)
 
 
-Quintessential Quintuplets
+Chuunibyou demo Koi ga Shitai! (Love, Chunibyo & Other Delusions!)
 ---
 
-### Nakano Nino
-
-![nino_dark code](../screenshots/quintuplets/nino_dark_code.png)
-
-### Nakano Miku
-
-![nakano_miku_dark code](../screenshots/quintuplets/nakano_miku_dark_code.png)
-
-### Nakano Yotsuba
+### Rikka Takanashi
```

Comment:
I'm going to switch this around because it's last name first name in [the master definition](https://github.com/doki-theme/doki-master-theme/blob/6bff67896d111a2f029f288f16efe8afa744d9f7/definitions/chuunibyou/rikka/dark/rikka.dark.master.definition.json#L3). Plus a large majority of the other themes are Last name, first name.